modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk.clj

```diff
@@ -279,107 +280,100 @@
   See https://cloud.google.com/bigquery/docs/querying-partitioned-tables#query_an_ingestion-time_partitioned_table"
   "_PARTITIONDATE")
 
-(defn- build-nested-column-lookup
-  "Returns a map of table-name->parent-path->nested-columns"
-  [driver database project-id dataset-id table-names]
+(defn- get-nested-columns-for-table
```

Comment:
An important question is whether doing this query table-by-table will have any significant cost implications